Documentation/Storage-Configuration/Advanced/ceph-configuration.md

```diff
@@ -23,6 +23,14 @@ to also change `ROOK_OPERATOR_NAMESPACE` to create a new Rook Operator for each
 forget to set `ROOK_CURRENT_NAMESPACE_ONLY`), or you can leave it at the same value for every
 Ceph cluster if you only wish to have one Operator manage all Ceph clusters.
 
+!!! note
+    If the operator namespace is different from the cluster namespace, the operator namespace must be
+	created before running the steps below. (The cluster namespace does not need to be created first,
```

Comment:
I think my suggestion to use parenthesis was not good, it's better to not use them. 
```suggestion
	created before running the steps below. The cluster namespace does not need to be created first,
```